docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -14,6 +14,8 @@ The bearer tokens are issued by OIDC and OAuth 2.0 compliant authorization serve
 
 To better understand OIDC Bearer authentication, see xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer authentication].
```

Comment:
@sberyozkin - I have a parallel task to fix references to Bearer authentication and make them consistent across all docs. Doing that today!

Reply:
Thanks @michelle-purcell